build/flaws.js

```diff
@@ -431,7 +437,22 @@ async function fixFixableFlaws(doc, options, document) {
             .replace(/\s+/g, "_")
             .toLowerCase()
         );
-        fs.writeFileSync(destination, imageBuffer);
+        // Before writing to disk, run it through the same imagemin
+        // compression we do in the filecheck CLI.
+        const compressedImageBuffer = await imagemin.buffer(imageBuffer, {
+          plugins: [getImageminPlugin(url.pathname)],
+        });
+        if (compressedImageBuffer.length < imageBuffer.length) {
+          console.log(
```

Comment:
Tiny super nit you can ignore. As this is informational, perhaps use `console.info`?

Reply:
We don't use `console.info` anywhere.

Reply:
Fair enough. I was thinking if we do, we could include that as something we allow with `eslint` line one sometimes allow `console.error`. As stated, I have no strong feelings here.